docs/docs/ncaa.md

```diff
@@ -134,7 +105,9 @@ Gets the scoreboard data for a specified date and team sport if available.
 
 **Example**  
 ```js
-const result = await sdv.ncaa.getScoreboard(
sport = 'basketball-men', division = 'd3', year = 2019, month = 02, day = 15
)
+const result = await sdv.ncaa.getScoreboard(
+sport = 'basketball-men', division = 'd3', year = 2019, month = 02, day = 15
```

Comment:
same re: above for all of these method examples. Not sure if that's coming from the docs auto-generation or something we can tweak manually.

Reply:
Yes this seems to be how the auto-generator works right now. As an experiment I added the indentation and then re-ran `npm run md-ncaa` and it changed the formatting back to this.